helm-chart/kuberay-apiserver/templates/service.yaml

```diff
@@ -6,6 +6,8 @@ metadata:
     prometheus.io/path: /metrics
     prometheus.io/scrape: "true"
     prometheus.io/port: "8888"
+  labels:
+    app.kubernetes.io/componen: kuberay-apiserver
```

Comment:
I think that `app.kubernetes.io/componen` should be `app.kubernetes.io/component`

Reply:
oh yes

Reply:
done